.circleci/config.yml

```diff
@@ -52,28 +67,58 @@ jobs:
               --filters "Name=platform,Values=windows" "Name=root-device-type,Values=ebs"
 
   integration-test-install-v1:
-    executor: aws-cli/default
+    parameters:
+      executor:
+        type: executor
+    executor: <<parameters.executor>>
     steps:
       - checkout
       - aws-cli/setup:
           version: "1"
       - run:
           name: Check latest v1 version is installed.
           command: |
-            aws --version
-            if [ "$(pip list --outdated | grep -c "awscli")" -ge 1 ]; then
-              echo "aws version is outdated"
+            which aws
+            if aws --version 2>&1 | grep -q 'aws-cli\/1.\d*.\d*'; then
+              echo AWS CLI v1 is installed
+            else
+              echo Wrong CLI version.
               exit 1
             fi
 
   integration-test-skip-install:
     # This image is suitable for the test as it comes with aws preinstalled
-    machine:
-      image: ubuntu-1604:201903-01
+    parameters:
+      executor:
+        type: executor
+    executor: <<parameters.executor>>
     steps:
-      - checkout
+      - aws-cli/install:
+          version: "2"
+      - aws-cli/setup:
+          version: "1"
+          skip-install-check: true
+
+  replace-pip-aws-test:
+    docker:
+      - image: cimg/python:3.8
+    steps:
+      - run: pip install aws
```

Comment:
I think you want https://pypi.org/project/awscli/

Reply:
Oh I see what you're doing - you're making the bad package install to get added to the path and then ensuring it gets reaplced correctly by the orb

Reply:
You got it